src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -33,16 +33,7 @@ export default function ProviderSelectionField({
         scrollAndFocus('#providerPostSelectionHeader');
       }
     },
-    [showProvidersList],
-  );
-
-  useEffect(
```

Comment:
This `useEffect` was used to focus on the `Choose a provider` button after selecting/removing a provider. Adding `providerSelected` to the `useEffect` above achieves the same effect and removes the extra focusing.